Jenkinsfile

```
@@ -946,7 +946,7 @@ pipeline {
                                                         name: 'Fault injection leaks',
                                                         id: 'NLT_client')]
                             junit testResults: 'nlt-junit.xml'
-                            archiveArtifacts artifacts: 'nlt_logs/centos7.fault-injection/'
+                            archiveArtifacts artifacts: 'nlt_logs/el8.fault-injection/'
```

You could probably make this more generic which will help with whatever the next upgrade is.  Alternatively neither this line nor L:928 need the disto in, they both just need to match.
```suggestion
                            archiveArtifacts artifacts: 'nlt_logs/'
```

---

I would like to leave that for a future PR.

---

Absolutely.